src/pages/react/index.md

```diff
@@ -9,4 +9,6 @@ It's declarative: React uses a declarative paradigm that makes it easier to reas
 
 To describe component's DOM representations, React uses an XML-like syntax called JSX. JSX is not required to use React, but it makes code more readable, and writing it feels like writing HTML. A simple transform is included with React that allows converting JSX into native JavaScript for browsers to digest.
 
-For more information head to [https://reactjs.org/](https://reactjs.org/)
\ No newline at end of file
+JSX is optional and not required to use React. So you can use ES6 and javascript's compiler like Babel. [https://babeljs.io/](https://babeljs.io/)
```

Comment:
The article currently states on Line 10 that "JSX is not required to use React" -- repeating this again on Line 12 seems redundant.

Reply:
Yep, you are right.